hbase-migration-tools/bigtable-hbase-replication/README.md

```diff
@@ -232,6 +214,57 @@ refer
 to [IncompatibleMutationAdapter](bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapter.java)
 javadocs for more details.
 
+### Dry run mode
+
+It may be hard to determine if an application issues incompatible mutations.
+Especially, if the HBase cluster and application are owned by different teams.
+The replication library provides a dry-run mode to detect incompatible
+mutations. In dry run mode, no mutations are sent over to Cloud Bigtable and
+replication never stalls. If you are not sure about incompatible mutations,
+enable replication in the dry run mode and observe the incompatible mutation
+metrics (discussed below).
```

Comment:
remove " and replication never stalls" in favor for something such as: 

In dry run mode, the Cloud Bigtable replication library is activated, but no mutations are sent over to Cloud Bigtable and check mutations for incompatibility. All mutations that are detected to be incompatible, will be logged ...

Reply:
PTAL.